http4s/client/src/main/scala/endpoints/http4s/client/Urls.scala

```diff
@@ -0,0 +1,140 @@
+package endpoints.http4s.client
+
+import cats.implicits._
+import endpoints.PartialInvariantFunctor
+import endpoints.Tupler
+import org.http4s.Uri
+import endpoints.Validated
+import org.http4s.Query
+import endpoints.algebra.Documentation
+import org.http4s.ParseResult
+
+trait Urls extends endpoints.algebra.Urls with StatusCodes {
+
+  type QueryString[A] = A => Query
+
+  type QueryStringParam[A] = A => List[String]
```

Comment:
Since `QueryStringParam` definitions are implicit, they will have the undesired effect of being eligible as implicit conversions with this type definition. I suggest using a type definition that is unrelated with `Function`